python/paddle/fluid/tests/unittests/dygraph_to_static/test_cinn_prim_mean.py

```diff
@@ -99,7 +100,12 @@ def _train(self, use_prim, data, axis, keep_dim):
     def check_prim(self, net, use_prim):
         if not use_prim:
             return
-        fwd_ops = [op.type for op in net.forward.main_program.block(0).ops]
+        fwd_ops = [
```

Comment:
leave a comment or seal this into a function, in case of possible changes happened later

Reply:
done